test/xrp-client-integration-test.ts

```diff
@@ -130,4 +130,14 @@ describe('Xpring JS XRPClient Integration Tests', function(): void {
     const doesExist = await xrpClient.accountExists(recipientAddress)
     assert.equal(doesExist, true)
   })
+
+  it('Payment History - rippled', async function(): Promise<void> {
+    this.timeout(timeoutMs)
+
+    const payments = await xrpClient.paymentHistory(recipientAddress)
+    // TODO(amiecorso): is this a sufficient integration test?
```

Comment:
We could check the length is greater than 1. Presumably the send test just ran above this one (which generated a payment transaction) so we should always get at least one result back.